### ISSUE TITLE
Get all vocab logits or prdicted token probability for MT5 model

### ISSUE THREAD

**Issue by ashleyLtj** [NVIDIA/FasterTransformer#640]
I want to use mt5-model for ranking task, and it needs to get the possibility of the predicted token.
In huggingface transformer, I can get each vocab logits and can use them to calculate softmax of the predicted token, the code is in https://github.com/huggingface/transformers/blob/v4.28.1/src/transformers/generation/utils.py
Can I get the same logits [batch_size, beam_width, vocab_size] in Faster-Transformer? If I need to modify the source code, which code should I modify?
